target_postgres/postgres.py

```diff
@@ -9,6 +9,8 @@
 import arrow
 from psycopg2 import sql
 
+import target_postgres.json_schema as JSONSchema
+
```

Comment:
This is one thing it pep 8 I kind of like but generally there are three import groups with a line in between:
```
built in packages/modules

3rd-party packages/modules

package internal packages/modules
```

```
import io
import re
import csv
import uuid
import json
from functools import partial
from copy import deepcopy

import arrow
from psycopg2 import sql

from target_postgres import json_schema
from target_postgres.singer_stream import (
    SINGER_RECEIVED_AT,
    SINGER_BATCHED_AT,
    SINGER_SEQUENCE,
    SINGER_TABLE_VERSION,
    SINGER_PK,
    SINGER_SOURCE_PK_PREFIX,
    SINGER_LEVEL
)
```

Reply:
Easily done!